windows-vm-configure-winrm/azuredeploy.parameters.json

```diff
@@ -0,0 +1,21 @@
+{
+    "$schema": "http://schema.management.azure.com/schemas/2015-01-01/deploymentParameters.json#",
+    "contentVersion": "1.0.0.0",
+    "parameters": {
+        "newStorageAccountName": {
+            "value": "testrg14"
```

Comment:
Can you checkout the guidance on [placeholder parameters](https://github.com/azure/azure-quickstart-templates#parameters-file-placeholders) so that this template can be CI friendly?